### PR TITLE
Fix multi-version installation instruction

### PR DESCRIPTION
**Pull request by alexxu-amd**
- Previous version used a loop to register the driver. It doesn't make sense since it is a overwrite operation. Change to use the latest version to register the driver
- Fixed an error in the rhel instruction ( name=ROCm|rocm_version|, only install the latest ROCm since rocm_version is static)
- Refactored SLES section for consistency

### CODE REVIEW

File: docs/install/native-install/includes/sles-multi-install.rst [hunk 1]
```diff
@@ -1,24 +1,56 @@
...
+              :sync: sles-{{ os_version }} sles-{{ os_major }}
+
+              .. code-block:: bash
+                  :substitutions:
+
+                  ver = |amdgpu_version|
+                  sudo tee /etc/zypp/repos.d/amdgpu.repo <<EOF
+                  [amdgpu]
+                  name=amdgpu
+                  baseurl=https://repo.radeon.com/amdgpu/$ver/sle/15.6/main/x86_64/
```
Comment (yhuiYH) [inline docs/install/native-install/includes/sles-multi-install.rst]:
```suggestion
                  baseurl=https://repo.radeon.com/amdgpu/$ver/sle/{{ os_version }}/main/x86_64/
```
I think you need this as a variable, not hard coded

File: docs/install/native-install/includes/rhel-multi-install.rst [hunk 2]
```diff
@@ -44,7 +43,7 @@
                  for ver in |rocm_multi_versions|; do
                  sudo tee --append /etc/yum.repos.d/rocm.repo <<EOF
                  [ROCm-$ver]
-                 name=ROCm|rocm_version|
+                 name=ROCm$ver
```
Comment (yhuiYH) [inline docs/install/native-install/includes/rhel-multi-install.rst:46]: good catch on this one!

**Review by yhuiYH — COMMENTED** [review]
Overall looks good.  Just that 1 fix, and I think we're good to merge.

**Comment by alexxu-amd** [comment]
Ram also wants Freddy to take a look before we publish. I'll ping him tomorrow

**Review by rponnuru5 — APPROVED** [review]
Changes made to fix the steps about latest kernel installation is good.

**Pull request merged**